submissions/zalevskyi/dom/js/app.js

```diff
@@ -0,0 +1,124 @@
+        context: 'A Castellan is a military officer responsible for directing the day-to-day affairs of a castle or fortress and the territory immediately around it. As castles were massive investments of treasure and important points of military power, the castellan had to be both competent and politically reliable. In times of war, a competent castellan could determine the fate of campaigns, and with that, the historical outcome.',
+        img_src: './img/Victor.png',
+        img_alt: 'Victor portrait'
+    }
+}
+Object.freeze(WIKI)
+const MAIN_INNER_HTML = `<input type='button' id='menu_button' class='menu_button' value='Menu'>
+                         <div id='civilopedia_path' class='civilopedia_path'>Civilization VI | Governors</div>
+                         <div id='article' class='article'><h1 id='title' class='text_heading_1'></h1>
+                            <img id='portrait' alt='' src='' class='portrait'>
+                            <h2 class='text_heading_2 description_heading'>Description</h2>
+                            <p id='description' class='text description'></p>
+                            <h2 class='text_heading_2 context_heading'>Historical Context</h2>
+                            <p id='context' class='text context'></p>
+                         </div>`
+const main = document.getElementById('main')
+const nav = document.getElementById('nav')
+main.innerHTML=MAIN_INNER_HTML
+const title = document.getElementById('title')
+const description = document.getElementById('description')
+const portrait = document.getElementById('portrait')
+const context = document.getElementById('context')
+const governorHeadings = document.querySelectorAll('.text_heading_2')
+hideGovernorDetails()
+updateMainCommon('governors')
+function updateMainCommon(name) {
+    title.textContent = `${name} - ${WIKI[name].title}`
+    description.textContent = WIKI[name].description
+}
+function updateMainGovernor(name) {
+    context.textContent = WIKI[name].context
+    portrait.src = WIKI[name].img_src
+    portrait.alt = WIKI[name].img_alt
+    showGovernorDetails()
+}
+function hideGovernorDetails() {
+    governorHeadings.forEach(element => element.classList.add('hidden'))
+    portrait.classList.add('hidden')
+    context.classList.add('hidden')
+    context.textContent = ''
+    portrait.src = ''
+    portrait.alt = ''
+}
+function showGovernorDetails() {
+    governorHeadings.forEach(element => element.classList.remove('hidden'))
+    portrait.classList.remove('hidden')
+    context.classList.remove('hidden')
+}
+function toggleHiddenSmallNavMain() {
+    main.classList.toggle('hidden_small')
+    nav.classList.toggle('hidden_small')
+}
+document.getElementById('menu').addEventListener('click', evt => {
+    const name = evt.target.textContent
+    const type = WIKI[name]['type']
+    document.querySelector('.menu_item_selected').classList.toggle('menu_item_selected')
```

Comment:
please use correct formatting. it helps to navigate your code easily. 
Also, I'd recommend starting to query this element not from the `document`, but just from the `evt.target`. And as you used `evt.target` multiple times here, you can create a constant here for storing the target value there.